utils/authenticationErrorAlert.js

```diff
@@ -0,0 +1,15 @@
+import { message as alert } from 'antd';
+
+export default (errMessage, error) => {
+  switch (errMessage) {
+    case 'No token provided':
+      alert.error('You need to login to rate an article');
+      break;
+    case 'Your account has not been verified, please verify to continue':
+      alert.error('Your account has not been verified, please verify to continue');
+      break;
+    default:
+      alert.error(error);
+      break;
+  }
+};
```

Comment:
This is a thoughtful abstraction, well done!

For the unverified user, instead of "please verify to continue", can it be "please verify to rate article". I think it will help a user when you are more specific